### PR TITLE
Feature/directions view

### PR DESCRIPTION
**Pull request by CloudNiner**

### CODE REVIEW

File: src/nih_wayfinding/app/scripts/routing/directions-service.js [hunk 1]
```diff
@@ -35,6 +37,38 @@
...
+            }
+        }
+
+        function getTurnIconName(turnType) {
+            // TODO: Write tests once actual icons exist
+            switch (turnType) {
+                case 'straight':
+                    return 'glyphicon-arrow-up';
+                case 'left':
+                case 'slightleft':
```
Comment (flibbertigibbet) [inline src/nih_wayfinding/app/scripts/routing/directions-service.js:62]: Might be good to have comments for the intentional case fall-throughs.

**Comment by flibbertigibbet** [comment]
+1

File: src/nih_wayfinding/app/scripts/routing/directions-service.js (hunk 1 above)
Comment (CloudNiner) [inline src/nih_wayfinding/app/scripts/routing/directions-service.js:62]: Sure, I'll add a note. Although, these are temporary until we get actual icons to display. Once that happens, I expect the slight case to be different.

File: src/nih_wayfinding/test/spec/views/routing/directions/directions-controller.spec.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+            }
+        };
+    }
+
+    beforeEach(inject(function ($controller, $q, $rootScope, RoutingResponseStub) {
+        rootScope = $rootScope;
+        scope = $rootScope.$new();
+        //Directions = _Directions_;
+        Map = mockMap(RoutingResponseStub);
+        //MapStyle = _MapStyle_;
```
Comment (kshepard) [inline src/nih_wayfinding/test/spec/views/routing/directions/directions-controller.spec.js:28]: Could remove these commented lines of code.

File: src/nih_wayfinding/app/scripts/routing/directions-service.js [hunk 3]
```diff
@@ -35,6 +37,38 @@
                 return RoutingResponseStub;
             }, 100, false);
         }
+
+        function getFlagIconName(flagType) {
+            // TODO: Write tests once actual icons exist
+            switch (flagType) {
+                case 'bench':
+                    return 'glyphicon-flash';
```
Comment (kshepard) [inline src/nih_wayfinding/app/scripts/routing/directions-service.js:45]: Careful, last time I used this icon as a placeholder, it never ended up getting replaced. Electrified benches :)

**Pull request closed without merging**